src/pages/Hojung/Login/Login.js

```diff
@@ -1,9 +1,99 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
 
 class LoginHojung extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      btn: false,
+    };
+  }
+
+  componentDidMount() {
+    const token = localStorage.getItem('token');
+    if (token) {
+      this.props.history.push('/main-hojung');
+    }
+  }
+
+  goToMain = e => {
+    e.preventDefault();
+    fetch('http://10.58.3.149:8000/users/signin', {
```

Comment:
```suggestion
    fetch(`${API}/users/signin`, {
```
물론 로그인 실습 때 사용한 코드이긴 하지만 이런 주소들은 src/config.js에 모아두면 좋습니다

Reply:
안녕하세요 ! 종택님 ! config.js 검색 중
config.js 파일로 분리하여 값을 export하고 전역변수(?)식으로 값을 넘겨주게 되면, 배포환경과 테스트 환경을 다르게 할 때 마다 변경해줘야하는 번거로움이 생긴다는 문제점을 알게 되었습니다.  다시 검색을 해보니 .env파일이
1.  development(개발모드)와 production (배포모드)두개를 나눠 작성 할 수 있기 때문에 편하다는 장점
2. 민감한 정보들이 변수 내 사용되는 것을 대비해 환경변수로 사용 되는 장점
이 있다는 것으로 이해하였습니다. 그래서 dotenv모듈을 이용해 env 파일로 분리해 보았습니다. (프로젝트 시작전 마지막 여유인 것 같아 이것저것 시도해보고 있는 중입니다 🔥 )
만약, 제 이해가 틀리거나 잘못된 부분이 있다면 말씀해주시면 감사하겠습니다 ~!

Reply:
대체로 맞게 이해하셨고, 전역 변수보다는 환경변수가 맞는 용어인 것 같습니다.
보통 .env 파일 안에 관련된 정보들을 담게 되는데, 2번의 경우에는 말씀해주신 이유 때문에 git에 올라가면 안됩니다. 그래서 .gitignore에 까지 추가해두고 파일은 별도의 방식으로 공유하는것이 옳습니다.